### PR TITLE
Throw exception if file size does not match.

### PR DESCRIPTION
**Pull request by mickel1138**

### CODE REVIEW

File: src/Util/FileUploader.php [hunk 1]
```diff
@@ -94,10 +96,16 @@ public function combineChunks(Request $request): array
...
         }
-        // Success
-        fclose($targetFile);
         rmdir($chunksFolder);
+        $targetFileSize = StreamInfo::getFileSize(array('fileStream' => $targetFile));
+        fclose($targetFile);
+        if ($targetFileSize !== $fileSize) {
+            unlink($targetFileName);
+            rmdir($targetDirectory);
+            throw new HttpException(Response::HTTP_BAD_REQUEST, "File size does not match!\n Expected file size is:$fileSize, and got $targetFileSize!");
```
Comment (praneethpr) [inline src/Util/FileUploader.php:105]: This message is not getting displayed in PROD mode

Comment (mickel1138) [reply src/Util/FileUploader.php:105]: This is going to be a problem, I'm going to look into this deeper. I cannot return a response here.

Comment (praneethpr) [reply src/Util/FileUploader.php:105]:
```suggestion
            throw new BadRequestHttpException("File size does not match!\n Expected file size is:$fileSize, and got $targetFileSize!");
```

Comment (praneethpr) [reply src/Util/FileUploader.php:105]: Maybe "Filesize doesn't match" is a good enough error message to the user

**Review by praneethpr — DISMISSED** [review]

File: src/Util/FileUploader.php [hunk 2]
```diff
@@ -94,10 +97,16 @@ public function combineChunks(Request $request): array
             fclose($chunk);
             unlink($chunksFolder . DIRECTORY_SEPARATOR . $i);
         }
-        // Success
-        fclose($targetFile);
         rmdir($chunksFolder);
+        $targetFileSize = StreamInfo::getFileSize(array('fileStream' => $targetFile));
+        fclose($targetFile);
+        if ($targetFileSize === $fileSize) {
```
Comment (mickel1138) [inline src/Util/FileUploader.php:103]:
```suggestion
        if ($targetFileSize !== $fileSize) {
```

Comment (mickel1138) [reply src/Util/FileUploader.php:103]: Only if they are NOT the same. ;-)

**Review by praneethpr — APPROVED** [review]

**Pull request merged**